src/nidmm/metadata/config.py

```diff
@@ -5,6 +5,7 @@
     'c_function_prefix': 'niDMM_',
     'driver_name': 'NI-DMM',
     'session_description': 'An NI-DMM session to a National Instruments Digital Multimeter',
+    'session_name': 'vi',
```

Comment:
Suggested renames for clarity:

* session_description  -> session_class_description
* session_name -> session_handle_parameter_name

Reply:
Changed